akka-actor/src/main/scala/akka/pattern/Patterns.scala

```diff
@@ -723,6 +723,21 @@ object Patterns {
       (attempted) => delayFunction.apply(attempted).toScala.map(_.toScala))(context, scheduler).asJava
   }
 
+  def retry[T](
```

Comment:
Add APIdoc like the other methods in here

Reply:
added